hack/release.yaml

```diff
@@ -28,15 +28,15 @@ SriovIbCni:
 Mofed:
   image: doca-driver
   repository: nvcr.io/nvstaging/mellanox
-  version: 24.01-0.3.3.1-0
+  version: 24.04-0.3.0.0-0
 RdmaSharedDevicePlugin:
   image: k8s-rdma-shared-dev-plugin
   repository: ghcr.io/mellanox
-  version: sha-fe7f371c7e1b8315bf900f71cd25cfc1251dc775
+  version: v1.4.0
```

Comment:
it should be `1.4.0-amd64`

Reply:
Done, switched to `1.4.0` as it's multiarch.